paper.md

```diff
@@ -129,18 +131,25 @@ Hack week outcomes, in turn, depend strongly on participants and are often a fun
+Thankfully, impostor syndrome can be mitigated. In minority participants, ensuring adequate representation among the participants can decrease feelings of otherness and thus reduce impostor syndrome. More generally, being open about its presence and prevalence can help participants feel more at ease (e.g. one Astro Hack Week participant remarked in a survey, _"I really appreciated the direct acknowledgement of impostor syndrome on the first day.  I think it helped ease the feeling!!"_) . Additionally, role models are very effective at encouraging positive behaviour: asking participants with prior experience at hack weeks at all academic levels to ask questions, even when they might know the answer, can help foster an inclusive environment that rewards risk taking. 
 
 
 ## Measures of success
 
-- See notes on IRB above; maybe expand on that here.
+Measuring the success of a hack week objectively is complicated by the diversity of goals of both organizing committee and attendees. A participant who attends primarily in order to hack on a specific project may find the tutorials less overall useful than a graduate student exposed to the tutorial topics for the first time. Additionally, the participant-driven, open-ended format facilitates knowledge transfer and collaborations in sometimes surprising ways that escape traditional measures of success. Additionally, it is difficult to define in the first place what success means to organizers and success. For more project-driven events, one key metric is the number of publications that result from hack week projects. However, this implies a very narrow definition of success in line with standard academic performance indicators, and may be hard to collect in practice, if participants do not explicitly include a reference to the event in their publication. Another, somewhat less traditional and complementary indicator is the activity of participants in terms of code written and committed to a public code repository (e.g. GitHub). This could be defined in lines of code committed, the number of new open-source projects spawned, and the number of new developers contributing to open-source projects. This assumes that participants work largely in the open during a hack week, and that most projects have a strong programming component. Again, this may not be true for some hacks (e.g. tutorials, other written text, visualizations), and focuses entirely on material outcomes. In particular, it ignores learning, community building as well as networking outcomes, which are often equally important objectives. One major approach here is to ask participants to self-report their success via a post-workshop survey. 
```

Comment:
A measure of success for geohack was the extent to which our existing tutorials were recycled by us in other venues (this can be explicitly measured), as well as the anecdotal evidence hearing from people who say they constantly refer back to our online content for reference.